test/test_callback_server_integration.py

```diff
@@ -265,6 +275,24 @@ def test_submit_job_sync(callback_ports):
         "version": "1.1"
     }
 
+    resp = _post(port, {"method":"CallbackServer.get_provenance"})
+    j = resp.json()
+    del j["result"][0][0]["time"]  # changes from run to run
+    assert j == {'result': [[{
+        'description': 'KBase SDK method run via the KBase Execution Engine',
+        'input_ws_objects': [],
+        'method': 'bar',
+        'method_params': [],
+        'service': 'foo',
+        'service_ver': 'beta',
+        'subactions': [{
+            'code_url': 'https://github.com/kbasetest/njs_sdk_test_1',
+            'commit': '366eb8cead445aa3e842cbc619082a075b0da322',
+            'name': 'njs_sdk_test_1',
+            'ver': '0.0.3-dev'
```

Comment:
Wait, it can't have a version and a dev at the same time, can it? I thought in order to have a version it must be beta or release?

Reply:
Whatever the version is in the kbase.yml file

Reply:
> For dev that's true, beta and release would have to have the same git commit I think. IIRC the catalog forces a version bump on any change

Nope, I'm wrong, you could move a new version to beta but not release